Fix broken auto-update as new manifest has not been successfully created.

The auto-update workflow failed with the following issue:

Run for version in 1.25.6 1.24.12; do
[main f6bb2ee] chore: generated nix manifest for go release 1.25.6
 1 file changed, 58 insertions(+)
 create mode 100644 manifests/1.25.6.nix
[main 626ea41] chore: generated nix manifest for go release 1.24.12
 1 file changed, 58 insertions(+)
 create mode 100644 manifests/1.24.12.nix
remote: Permission to purpleclay/go-overlay.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/purpleclay/go-overlay/': The requested URL returned error: 403
Error: Process completed with exit code 128.

--- a/.github/workflows/auto-update.yml
+++ b/.github/workflows/auto-update.yml
@@ -11,6 +11,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6
+        with:
+          token: ${{ secrets.GH_NSV }}
 
       - name: Install Nix
         uses: cachix/install-nix-action@4e002c8ec80594ecd40e759629461e26c8abed15 # v31
@@ -42,15 +44,11 @@ jobs:
             fi
           done
 
-      - name: Import GPG key
-        if: ${{ env.MANIFESTS_GENERATED == 'true' }}
-        uses: crazy-max/ghaction-import-gpg@e89d40939c28e39f97cf32126055eeae86ba74ec #v6
-        with:
-          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
-          passphrase: ${{ secrets.GPG_PASSPHRASE }}
-          trust_level: 5
-          git_user_signingkey: true
-          git_commit_gpgsign: true
+      - name: GPG Import
+        env:
+          GPG_PRIVATE_KEY: "${{ secrets.GPG_PRIVATE_KEY }}"
+          GPG_PASSPHRASE: "${{ secrets.GPG_PASSPHRASE }}"
+        run: nix run github:purpleclay/gpg-import
 
       - name: Commit and Push Manifests
         if: ${{ env.MANIFESTS_GENERATED == 'true' }}

